test/jdk/java/nio/file/Path/PathOps.java

```diff
@@ -1669,6 +1706,26 @@ static void doUnixTests() {
             .resolve("foo", "foo")
             .resolve("/foo", "/foo");
 
+        // resolve - varargs
+        test("/tmp")
+            .resolve("/tmp/foo/bar/gus", "foo", "bar", "gus")
+            .resolve("/gus", "/foo", "bar", "/gus")
+            .resolve("/tmp/baz", "", "", "baz");
+        test("tmp")
+            .resolve("tmp/foo/bar/gus", "foo", "bar", "gus")
+            .resolve("/gus", "/foo", "bar", "/gus")
+            .resolve("tmp/baz", "", "", "baz");
+        test("")
+            .resolve("", "", "")
+            .resolve("/bar", "foo", "/bar", "")
+            .resolve("foo/bar/gus", "foo", "bar", "gus")
+            .resolve("baz", "", "", "baz");
+        test("/")
+            .resolve("/foo", "", "", "foo", "")
+            .resolve("/foo", "foo", "")
+            .resolve("/foo", "", "foo")
+            .resolve("/bar", "foo", "", "/bar");
+
```

Comment:
Should there be test cases that include multiple directories in a path:  "abc/def/xyz"?
Also cases with "..", or is that only relevant if canonicalizing.

Reply:
Multiple directories would be good but I don't know about `..`.

Reply:
> Should there be test cases that include multiple directories in a path: "abc/def/xyz"?

Added some multiple directory cases in 505d72b48c5d34bbf4474c6fed43e2f269e8e135.

> Also cases with ".."

This is not relevant. For example, using the existing `resolve(Path)`:

```
jshell> java.nio.file.Path.of("/tmp").resolve("..").resolve("..")
$1 ==> /tmp/../..
```

Reply:
Addressed in c2a8faaa1b81dbb40e1a5989a02c9e5935417358.